src/manager.cpp

```diff
@@ -190,11 +190,19 @@ sdbusplus::async::task<>
         watch::inotify::DataWatcher dataWatcher(
```

Comment:
Commit message:

Header: `Manager: Update disable sync with latest code` 

Please avoid this type of commit message header. We've discussed this pattern previously as well. Please describe what are you doing in the commit message header and why should be in commit message body as per guideline (https://github.com/openbmc/docs/blob/master/CONTRIBUTING.md#submitting-changes). 

> - Moved PeriodicDisablePropertyTest from manager_test to
  periodic_sync_test to avoid conflicts with common JSON file config.

It should be in separate commit and why are we moving need to capture.

> - Added 'testDataChangeDisableProperty' test case.
Not required to capture UT cases details.

> - Tested on simics.

Please elaborate like how do you test in the simics.